src/core/thread/network_data_local.cpp

```diff
@@ -195,13 +196,33 @@ ThreadError Local::UpdateRloc(BorderRouterTlv &aBorderRouter)
     return kThreadError_None;
 }
 
+bool Local::IsOnMeshPrefixConsistent(void)
+{
+    return (mLeader.ContainsOnMeshPrefixes(*this, mMle.GetRloc16()) &&
+            ContainsOnMeshPrefixes(mLeader, mMle.GetRloc16()));
+}
+
+bool Local::IsExternalRouteConsistent(void)
+{
+    return (mLeader.ContainsExternalRoutes(*this, mMle.GetRloc16()) &&
+            ContainsExternalRoutes(mLeader, mMle.GetRloc16()));
+}
+
 ThreadError Local::SendServerDataNotification(void)
 {
     ThreadError error = kThreadError_None;
     uint16_t rloc = mMle.GetRloc16();
 
```

Comment:
... seems it might be more reasonable to bundle `prefix` and `netdataregister` or unbundle them by distinguishing `configBorderRouter()` after or before `thread start`. 
Will confirm with @xiaom-GitHub about this THCI API implementation.
Sorry, please keep this as it is now.